### PR TITLE
Add Travis-CI support

### PR DESCRIPTION
**Pull request by AdamBJ**
I'm not looking to merge this yet. This PR is to test Travis CI.

### CODE REVIEW

**Comment by AdamBJ** [comment]
The problem is that Travis can't find the local `umple-latest.jar`. It would be much better if we had that jar in a maven repo... any workaround I apply to make the jar available locally is going to have to be removed once we get the maven repo up and running.

**Comment by AdamBJ** [comment]
Well, the good news is that I fixed the download task. I think it was broken before. So now the download correctly kicks off if there's a new version of the umple jar. The problem is that it won't download anything if there isn't any jar at all. Perhaps until we get the maven setup we can check in an umple jar? It's only 5mb, and we'd always be using the most up-to-date one thanks to the download task fix. @vahdat-ab @Nava2  what do you guys think? It's a ugly workaround, but hopefully we'll get the maven repo up soon.

File: build.gradle [hunk 1]
```diff
@@ -45,6 +45,15 @@ apply plugin: 'de.undercouch.download'
 apply plugin: "com.gradle.plugin-publish"
 apply plugin: 'maven'
 
+task downloadUmpleJar(type: de.undercouch.gradle.tasks.download.Download) {
```
Comment (Nava2) [inline build.gradle:48]: Could we do this via travis for now? Use `wget` or something?

**Comment by AdamBJ** [comment]
@Nava2 @vahdat-ab This PR should be good to go now. It adds Travis-CI support for the plugin. Travis first `wget`s the umple jar then runs the build task, which in turn runs all the integration and unit tests. This PR also fixes the download task, a task which, as it turns out, isn't necessary to get Travis working. But I thought it was when I was putting this PR together, so the fix is included in it.

resolves #4

**Comment by Nava2** [comment]
:+1:

**Pull request merged**